### PR TITLE
Splash sidenav updates

### PR DESCRIPTION
**Pull request by gSoD17**

### CODE REVIEW

**Comment by domdreams** [comment]
Hey @gSoD17 this looks amazing! My only question is, would it be hard to have the district underlined when its expanded like this 
Screen Shot 2020-11-18 at 12 25 16 PM

It is already a hover state but wondering if it can be underlined when expanded too. Otherwise it looks great!

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
         <ng-container *ngFor="let candidateType of sortedObj | keyvalue: asIsOrder; let i= index">
 
-          <mat-expansion-panel (opened)="setOfficeStep(i)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
+          <mat-expansion-panel [expanded]="officeType === candidateType.key ? true : false" (opened)="setOfficeStep(i); setOfficeType(candidateType.key)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
```
Comment (ShrimpMantis) [inline src/app/components/home/home.component.html:18]: we can shorten it, officeType===candidateType.key

Comment (gSoD17) [reply src/app/components/home/home.component.html:18]: Cool, will go and shorten it.

**Review by robertgz — APPROVED** [review]
Looks Good!

**Pull request merged**